CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/AnalyticConfigurationPane.java

```diff
@@ -143,7 +143,7 @@ public AnalyticConfigurationPane() {
                 .map(plugin -> plugin.getPlugin().getResultType())
                 .collect(Collectors.toSet()).size() <= 1) :
                 "A category should be populated only with analytics of the same result type.";
-        this.categoryList = new ListView();
+        this.categoryList = new ListView<String>();
```

Comment:
I'd assume this can just be `ListView<>`?

Reply:
Yep will be fixed in the next commit